virttest/qemu_vm.py

```diff
@@ -547,6 +547,11 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                 if ctrl_mac_addr and ctrl_mac_addr in ["on", "off"]:
                     dev.set_param('ctrl_mac_addr', ctrl_mac_addr)
                 dev.set_param('mac', mac, dynamic=True)
+            if "virtio" in model:
+                if int(queues) > 1:
+                    dev.set_param('mq', 'on')
+                if vectors:
+                    dev.set_param('vectors', vectors)
```

Comment:
I don't think this is the right position for this `if`, cause previously it is:
```
if devices.has_option("device"):
    ''' General handling when there is qemu support '-device''''
else:
    '''Use -net when -device not supported'''
```
While now you change it to:
```
if devices.has_option("device"):
    ''' Part of general handling when there is qemu support '-device' '''
if  "virtio" in model:
    '''Another part of general handling when there is qemu support '-device', including some non-'virtio' handling like  'spapr-vlan' '''
else:
    '''Use -net when -device not supported'''
```
It will hurt the generation of other nic model type besides 'virtio'.

Reply:
I have solved this issue,please help me review it again,thanks.